pkg/generator.go

```diff
@@ -77,6 +85,11 @@ func (g Generator) Create() (string, error) {
 				log.Fatal(err)
 			}
 			name := pod.Name
+			if len(name) != 0 {
+				if (len(skipObj["Pod"]) != 0) && (skipObj["Pod"][0] == name) {
```

Comment:
I think we should support passing namespace optionally with the --skip-resources, since the other flags for Chartify support specifying namespace optionally. So, valid values for the flag can be:
```
--skip-resources=Pod/ngnix
--skip-resources=Pod/ngnix@namespace
```
To extract namespace, just reuse `pkg/kube_objects.go#splitNamespace` method. So, call `splitNamespace(skipObj["Pod"][0])` and match against the `pod.Namespace` in addition to `pod.Name` before skipping.